storage/persistent_storage/rosa-persistent-storage-aws-ebs.adoc

```diff
@@ -0,0 +1,70 @@
+:context: rosa-persistent-storage-aws-ebs
+
+toc::[]
+
+{product-title} (ROSA) clusters are pre-built with two storage classes that use AWS Elastic Block Store (EBS) volumes. These storage classes are ready to use and some familiarity with Kubernetes and AWS is assumed.
+
+Following are the two pre-built storage classes:
+[options="header"]
+
+|===
+
+| Name | Provisioner
+
+| gp2 | kubernetes.io/aws-ebs
+
+| gp2-csi | ebs.csi.aws.com
+
+|===
+
+[NOTE]
+====
+The gp2 storage class is set as default; however, you can select either one as the default storage class.
+====
+
+The Kubernetes persistent volume framework enables the administrators to provision a cluster with persistent storage and gives users a way to request those resources without having any knowledge of the underlying infrastructure. You can dynamically provision the AWS EBS volumes. Persistent volumes are not bound to a single project or namespace; therefore, the volumes can be shared across ROSA clusters. Persistent volume claims are specific to a project or namespace and can be requested by users.
+
+[IMPORTANT]
+====
+ROSA defaults to using an in-tree or non-Container Storage Interface (CSI) plug-in to provision AWS EBS storage. In future ROSA versions, volumes provisioned using existing in-tree plug-ins are planned for migration to their equivalent CSI driver. After full migration, the in-tree plug-ins will eventually be removed from the future versions of ROSA.
+====
+
+[IMPORTANT]
+====
+High-availability of storage in the infrastructure is left to the underlying storage provider.
+====
+
+== Format of persistent volumes
+Before a ROSA cluster mounts the volume and passes it to a container, the cluster checks that the volume contains a file system as specified by the *fsType* parameter in the persistent volume definition. If the device is not formatted with the file system, all data from the device is erased and the device is automatically formatted with the given file system.
+This verification enables you to use unformatted AWS volumes as persistent volumes, as the ROSA cluster formats the AWS volumes before the first use.
+
+
+== Capacity of EBS volumes on a node
+By default, a ROSA cluster supports a maximum of 39 EBS volumes attached to one node. This limit is consistent with the link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/volume_limits.html#linux-specific-volume-limits[AWS volume limits]. The volume limit depends on the instance type.
+
+[IMPORTANT]
+====
+You must use either in-tree or CSI volumes and their respective storage classes, but never both volume types at the same time. The maximum attached EBS volume number is counted separately for in-tree and CSI volumes.
+====
+
+== Creating a persistent volume claim
+
+.Prerequisites
+Storage must exist in the underlying infrastructure before it can be mounted as a volume in the ROSA cluster.
+
+.Procedure
+. In the OpenShift Cluster Manager (OCM) console, click *Storage → Persistent Volume Claims*.
```

Comment:
It should be OpenShift cluster console rather than 'OpenShift Cluster Manager (OCM) console'.

Reply:
Made the change, thanks.